app/src/pages/inside/logsPage/sauceLabsSection/jobInfoSection/logsContent/logsContent.jsx

```diff
@@ -2,15 +2,12 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import { sauceLabsLogsSelector } from 'controllers/log/sauceLabs';
-import styles from './logsSection.scss';
+import styles from './logsContent.scss';
 
 const cx = classNames.bind(styles);
 
-@connect((state) => ({
-  logs: sauceLabsLogsSelector(state),
-}))
-export class LogsSection extends Component {
+@connect(null)
```

Comment:
?

Reply:
Just a template for future component. Will be fixed in the next PR